test/integration/deploy.test.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+const chai = require('chai');
+
+const { fetch } = require('../utils/integration');
+const runServerless = require('../run-serverless');
+
+chai.use(require('chai-as-promised'));
+chai.use(require('sinon-chai'));
+
+const { expect } = require('chai');
+
+const clientConfigs = [
+  { bucketName: 'sls-finch-test-stndrd-use1-v3', region: 'us-east-1' },
+  { bucketName: 'sls-finch-test-stndrd-usw2-fer', region: 'us-west-2' }
+];
```

Comment:
Bucket names and regions are hardcoded currently, so will fail unless you change those values. Probably we should change them to be environment vars as well. 

I also think (but need to double check) that the buckets must exist already). Assuming this library doesn't create buckets for you.

Wondering if we should try to create the buckets automatically, or if that's overkill.

Reply:
I don't know how hard it would be but I would create a new bucket every time with a new name to make external devs able to run tests without much setup

here's my suggestions:

1. Through cli run a command to get the aws account number `aws sts get-caller-identity --query "Account" --output text`
2. create 2 buckets named 'sls-finch-test-stndrd-use1-v3'+accNumber, sls-finch-test-stndrd-usw2-fer'+accNumber
3. runTests
4. Empty and remove buckets with the `aws s3 rb --force` command 
Step 4 needs to happen whether it's a success or a failure
maybe through try{}finally{} or something similar
it should also be easy to bypass step 4 in order to debug the issue and look at the bucket

Reply:
We need to have the account number in the bucket so that the bucketname is unique for every account running the tests otherwise s3 will complain about the bucket name not being unique

Reply:
Good ideas, I'll look into this tomorrow hopefully.  Any reason to use the account I'd when creating a bucket (vs a UUID or something similar)?

Reply:
UUID would be fine too. but if we are failing to delete the bucket we would be hitting the S3 limit of bucket count 
so randomizing the bucket name would be a bad idea

this way worst case scenario we would be always overriding the same bucket and not taking up so many buckets from the account quota

[By default, you can create up to 100 buckets in each of your AWS accounts. If you need additional buckets, you can increase your account bucket limit to a maximum of 1,000 buckets by submitting a service limit increase. There is no difference in performance whether you use many buckets or just a few.
](https://docs.aws.amazon.com/AmazonS3/latest/userguide/BucketRestrictions.html#:~:text=By%20default%2C%20you,just%20a%20few.)

Reply:
I've updated the integration tests so that the bucket names use the account ID, and buckets are automatically deleted before and after the tests.

I also added a README to the test directory

Reply:
I will review this sometime this weekend thanks so much